### DISCUSSION TITLE
Is there an easy way to tell the unit of metrics?

### DISCUSSION THREAD

**Discussion by jubingc** [milvus-io/milvus#37463] [Q&A and General discussion] [answered]
Hi,

I am viewing milvus metrics from https://milvus.io/docs/metrics_dashboard.md. I am wondering about units for those latency metrics especially because they are not specified anywhere I can find. Besides, those latency metrics do not share the same unit. This increases the difficulties when explaining metrics.

For example, proxy_req_latency [uses milliseconds](https://github.com/milvus-io/milvus/blob/b2c36efb5feb4ee48047602ebc00dfa33dbe54cb/internal/proxy/impl.go#L465), while indexnode_save_index_latency [uses seconds](https://github.com/milvus-io/milvus/blob/b2c36efb5feb4ee48047602ebc00dfa33dbe54cb/internal/indexnode/task_index.go#L360) and datacoord_task_execute_max_latency [uses nanosecond](https://github.com/milvus-io/milvus/blob/b2c36efb5feb4ee48047602ebc00dfa33dbe54cb/internal/datacoord/task_scheduler.go#L363).

**Comment by yhmo** [comment]
The first paragraph of the doc https://milvus.io/docs/metrics_dashboard.md mentions "The time unit in this topic is milliseconds. And “99th percentile” in this topic refers to the fact that 99 percent of the time statistics are controlled within a certain value."

**Reply by LoveEachDay** [reply, marked as answer]
@jubingc Most of the latency related metrics use milliseconds while index related metrics use seconds instead.
`proxy_req_latency` use milliseconds, `datacoord_task_execute_latency` use milliseconds, while `indexnode_save_index_latency` use seconds. 

We'll add a column to describe the metric unit in the https://milvus.io/docs/metrics_dashboard.md.

**Reply by jubingc** [reply]
Thank you for all of your responses.
